apps/src/aiTutor/chatApi.ts

```diff
@@ -8,8 +8,25 @@ import {
 import HttpClient from '@cdo/apps/util/HttpClient';
 import Lab2Registry from '../lab2/Lab2Registry';
 
+// These are the possible statuses returned by ShareFiltering.find_failure
+enum ShareFilterStatus {
+  Email = 'email',
+  Phone = 'phone',
+  Address = 'address',
+  Profanity = 'profanity',
+}
+
 const CHAT_COMPLETION_URL = '/openai/chat_completion';
 
+// Analogous to https://github.com/code-dot-org/ml-playground/pull/299
+// We want to expose enough information to help troubleshoot false positives
+const logViolationDetails = (response: OpenaiChatCompletionMessage) => {
+  console.info('Violation detected in chat completion response', {
```

Comment:
should we log this to cloudwatch instead? Or in addition to this?

Reply:
Great idea. It'd be nice to not have to wait for users to report these to see how aggressive it's being. Thanks, Molly!

Reply:
seems reasonable to log a generic event for now!